Document enroll secret activities in audit logs (#37418)
fleetdm/fleet: docs/Contributing/reference/audit-logs.md

}
```

## modified_enroll_secret

> Generated when an enroll secret is modified (updated, added, or deleted).

Reviewer: I think the add/update/delete phrasing matches what we do in the UI but is less clear from an API perspective. 
```suggestion
Generated when global or team enroll secrets are modified.
```
